vim-patch:8.2.{3684,3687}: blockwise insert does not handle autoindent properly

clason opened the pull request:
Problem:    Blockwise insert does not handle autoindent properly.
Solution:   Adjust text column for indent. (closes vim/vim#9229)
https://github.com/vim/vim/commit/e9b0b40b7978f683977922233b42dd439ef31920

Problem:    Blockwise insert does not handle autoindent properly when tab is inserted.
Solution:   Adjust text column for indent before computing column.
            (closes vim/vim#9229)
https://github.com/vim/vim/commit/59f4f9505ae7ca2499904b94100db103e5ada5a6

clason committed:
vim-patch:8.2.3684: blockwise insert does not handle autoindent properly

Problem:    Blockwise insert does not handle autoindent properly.
Solution:   Adjust text column for indent. (closes vim/vim#9229)
https://github.com/vim/vim/commit/e9b0b40b7978f683977922233b42dd439ef31920

zeertzjq commented on src/nvim/ops.c:
```diff
@@ -2311,6 +2311,13 @@ void op_insert(oparg_T *oap, long count1)
         }
       }
       bd.textcol = bd2.textcol;
+      /*
+       * If the insert was in the indent then include the indent
+       * change in the new text, otherwise don't.
+       */
+      if (did_indent && bd.textcol > ind_pre) {
+          bd.textcol += ind_post - ind_pre;
+      }
```
```suggestion
      if (did_indent && bd.textcol > ind_pre) {
          // If the insert was in the indent then include the indent
          // change in the new text, otherwise don't.
          bd.textcol += ind_post - ind_pre;
      }
```

clason replied:
Man, did I get the style guide wrong 🤦🏻 

(Sorry, @dundargoc !)

zeertzjq commented:
There is another patch with the same message
<https://github.com/vim/vim/commit/59f4f9505ae7ca2499904b94100db103e5ada5a6>

clason committed:
vim-patch:8.2.3687: blockwise insert does not handle autoindent properly

Problem:    Blockwise insert does not handle autoindent properly when tab is
            inserted.
Solution:   Adjust text column for indent before computing column.
            (closes vim/vim#9229)
https://github.com/vim/vim/commit/59f4f9505ae7ca2499904b94100db103e5ada5a6
